### PR TITLE
icon rework

### PR DESCRIPTION
**Pull request by myangelkamikaze**

### CODE REVIEW

File: ElectronicObserver/Window/Wpf/Fleet/ViewModels/FleetStateViewModel.cs [hunk 1]
```diff
@@ -407,14 +407,12 @@ public void RefreshFleetState()
 
 		}
 	}
-	public int GetIconIndex()
+	public IconContent GetIconIndex()
```
Comment (Jebzou) [inline ElectronicObserver/Window/Wpf/Fleet/ViewModels/FleetStateViewModel.cs:410]: Should be renamed GetIcon since we're not getting the index anymore ?

Comment (myangelkamikaze) [reply ElectronicObserver/Window/Wpf/Fleet/ViewModels/FleetStateViewModel.cs]: Yeah, makes sense.
Wonder if the IconContent enum should be IconId or something instead, think that would be clearer.

File: ElectronicObserver/FormMainWpf.xaml
Comment (Jebzou) [inline ElectronicObserver/FormMainWpf.xaml:1]: "View Dropdown" seems to display the icons with some kind of default size, is there a way to force it to 16x16 ? 
image

Comment (Jebzou) [reply ElectronicObserver/FormMainWpf.xaml:1]: Might be related, I'm getting those binding errors 
image

Comment (myangelkamikaze) [reply ElectronicObserver/FormMainWpf.xaml:1]: Think it's unrelated to that. The reason the icons are blurry is that they're getting stretched. With the old code they were hard-coded to 16 physical pixels, they should be 16 logical pixels now.

**Review by Jebzou — APPROVED** [review]

**Pull request merged**